src/coreclr/src/utilcode/longfilepathwrappers.cpp

```diff
@@ -768,28 +768,15 @@ FindFirstFileExWrapper(
 
 #ifdef HOST_WINDOWS
 
-#if ! defined(DACCESS_COMPILE) && !defined(SELF_NO_HOST)
-extern HINSTANCE            g_hThisInst;
-#endif// ! defined(DACCESS_COMPILE) && !defined(SELF_NO_HOST)
-
 BOOL PAL_GetPALDirectoryWrapper(SString& pbuffer)
```

Comment:
Do we still need `PAL_GetPALDirectory` and all its friends? It seems that it can be just replaced with `GetClrModuleDirectory` everywhere.

Reply:
Possibly. `PAL_GetPALDirectoryW` has a lot of uses. There is also an overload that takes two arguments. I will take a look.

Reply:
Searching for `PAL_GetPALDirectory` has many hits, but I think most of them are in tests, etc. There are very few actual uses as far as I can tell.

Reply:
Using GetClrModuleDirectory here